tensorboard/uploader/proto/export_service.proto

```diff
@@ -80,6 +85,19 @@ message StreamExperimentsResponse {
   repeated Experiment experiments = 2;
 }
 
+// Request to get the metadata of an experiment.
+message GetExperimentMetadataRequest {
+  // ID of the experiment to get.
+  string experiment_id = 1;
+}
```

Comment:
Let’s include an `experiment_mask` here so that we can avoid requesting
some of the more expensive fields if we don’t need to?